app/router.js

```diff
@@ -1,7 +1,16 @@
 'use strict';
 
+const osLocale = require('os-locale');
+const locale = osLocale.sync();
+// China zh_CN
+const defaultLocale = locale === 'en_US' ? '/en': '/zh-cn';
```

Comment:
I guess that is not what I was trying to do. It should handle at the Client side.